pkg/resource/aws/aws_appautoscaling_policy_test.go

```diff
@@ -0,0 +1,31 @@
+package aws_test
+
+import (
+	"testing"
+
+	"github.com/cloudskiff/driftctl/test"
+	"github.com/cloudskiff/driftctl/test/acceptance"
+)
+
+func TestAcc_Aws_AppAutoscalingPolicy(t *testing.T) {
+	acceptance.Run(t, acceptance.AccTestCase{
+		TerraformVersion: "0.15.5",
+		Paths:            []string{"./testdata/acc/aws_appautoscaling_policy"},
+		Args:             []string{"scan", "--filter", "Type=='aws_appautoscaling_policy'"},
+		Checks: []acceptance.AccCheck{
+			{
+				Env: map[string]string{
+					"AWS_REGION": "us-east-1",
+				},
+				Check: func(result *test.ScanResult, stdout string, err error) {
+					if err != nil {
+						t.Fatal(err)
+					}
+					result.AssertInfrastructureIsInSync()
+					result.AssertManagedCount(1)
+					result.AssertDriftCountTotal(0)
```

Comment:
No need for this line since it is implicit inside the `AssertInfrastructureIsInSync`